controllers/dbaas.redhat.com/inventory.go

```diff
@@ -19,6 +19,7 @@ const (
 	CPU           = "cpu"
 	MEMORY        = "memory"
 	IS_HA         = "is_ha"
+	CLUSTER_NAME  = "name"
```

Comment:
Should phase be added to the inventory instance info as per the ADR?

Reply:
need to update the ADR - see the current implementation : https://github.com/RHEcosystemAppEng/dbaas-operator/blob/4494a452c155eb4318ca4a854451f4de4c83da61/api/v1alpha1/dbaasprovider.go#L223

Reply:
@jianrongzhang89 I see what you mean added the `state into instance info to get cluster state